src/lib/yang/snabb-softwire-v2.yang

```diff
@@ -757,6 +771,7 @@ module snabb-softwire-v2 {
 
       list softwire {
         key "ipv4 psid";
+        unique "b4-ipv6";
```

Comment:
I see why this is added (ieee-softwire constraints) but I am not sure that it's the right thing.  It's not a constraint from the perspective of `apps.lwaftr`.  Considering the data structure that the lwaftr uses, it would imply that adding/removing softwires using the native schema become an O(n) operation.

The problem that this is preventing is that requests in terms of ieee-softwire will choose a snabb-softwire-v2 softwire with matching ipv6 address in an arbitrary way.  I would just document this restriction and be done with it, and not enforce it via the schema.  Would that be OK for you?

Reply:
Reverted and replaced by a warning in 791a8a9